app/api/views/users.py

```diff
@@ -98,6 +99,7 @@ def login():
             session.regenerate()
         login_user(user)
 
+        user = load_user(user.id)
```

Comment:
This was done because the user data being returned on login previously did not include the special properties added by the `User` class we define in the dmutils library, such as the `teams` property. This is needed because the buyer frontend will redirect users to the 'join a team' page on login, where relevant.